Sources/TSCUtility/Tracing.swift

```diff
@@ -135,8 +135,24 @@ public struct TracingEvent: TracingEventProtocol, Codable {
     #endif
 }
 
-public class TracingCollection: TracingCollectionProtocol {
-    public var events: [TracingEventProtocol] = []
+public final class TracingCollection: TracingCollectionProtocol, @unchecked Sendable {
```

Comment:
@unchecked because of NSLock? (if so, can add a comment?) Do we have a Lock implementation in TSC which _is_ sendable, that we could use instead?

Reply:
It's unchecked because it's manually synchronised with the lock. Even if the lock itself is sendable, the compiler can't check that you've used it correctly.

The only way to get that is to use something like `NIOLockedValueBox<[TracingEventProtocol>]` but that's not available here.